doc/manual/manual.tex

```diff
@@ -8485,23 +8488,249 @@ \subsubsection{The van Keken thermochemical composition benchmark}
 
 We can observe that when the two-layer viscosity model is added, there is only one apparent peak for each case. The first peaks of the 0.01 viscosity contrast tests appear earlier and are larger in magnitude than those of 0.1 viscosity contrast tests.  There are no secondary plumes and the whole system tends to reach stability after around 500 time steps.
 
+
+\paragraph{Computation of the van Keken Problem with the Volume-of-Fluid Interface Tracking Method}
+\label{paragraph:van-keken VOF interface tracking}
+
+One can also model the van Keken problem with the volume-of-fluid (VOF) interface tracking 
+algorithm in ASPECT.
+In fact, this problem is particularly well-suited to being computed with the VOF method, since 
+it consists of two distinct, immiscible fluids and interface tracking algorithms are 
+specifically designed not to allow the two fluids to mix.
+In particular, assuming the computation is sufficiently well-resolved, the fluids will not mix 
+at sub-grid scales over the entire duration of the computation.
+However note that this implies that all computations of the van Keken problem made with the VOF 
+method must necessarily be with discontinuous initial conditions.
+Finally, since one is often interested in a high resolution image of the shape of the interface 
+between the two fluids, one can use the output of the VOF method to examine the interface 
+within individual cells or within regions consisting of groups of cells.
+
+Another advantage the VOF method has over modeling moving interface problems with compositional 
+fields is that for problems in which the interface occupies a relatively small part of the 
+computational domain, all of the computational work in the VOF method is done in cells that lie 
+in a neighborhood of the interface, rather than in the entire computational domain. 
+
+\begin{figure}[b!]
+   \centering
+\includegraphics[width=0.5\textwidth]{cookbooks/benchmarks/vof-van-keken/rms_vel_comparison.png}
```

Comment:
To my eye, I can really only see the lines for refinements 5-7. Perhaps this is all that is needed to show convergence?

Reply:
I would appreciate it @egpuckett would comment. It is true that after 7, the lines are overlapping to visual examination on the plot in question, but not including the further lines would make including that point more questionable.

Reply:
Please keep all lines / levels of refinement in.  Unless I missed something we are duplicating exactly what was done in 

> 5.4.3 The van Keken thermochemical composition benchmark

> This section is a co-production of Cedric Thieulot, Juliane Dannberg, Timo Heister and Wolfgang Bangerth with an extension to this benchmark provided by the Virginia Tech Department of Geosciences class “Geodynamics and ASPECT" co-taught by Scott King and D. Sarah Stamps.

We want to compare Apples to Apples.  We think VOF is a much better methodology **for certain problems**, but there seem to be a lot of, or maybe only some influential, ASPECT users and computational geodynamicists that think otherwise and would just as soon this methodology - interface tracking - went away.  Personally, I think that is so crazy that I can't find words to express how backward thinking this is.  Interface tracking is the right way to do certain problems in every field of mechanics, fluid mechanics, not to mention a gazillion other examples.  We need the support of the ASPECT community to implement a 3D VOF method and one that works on any kind of arbitrary grid that is supported by deal.ii.

I think the way to do that is to show everyone how much better the VOF results are **on some problems**, and this one in particular.